scripts/checkout_web_vault.sh

```diff
@@ -47,10 +48,15 @@ else
     popd
 fi
 
+CHECKOUT_ARGS=""
+if [ "$CHECKOUT_TAGS" = "true" ]; then
+    CHECKOUT_ARGS="$CHECKOUT_ARGS --tags"
+fi
+
 # Checkout the request
 pushd "${VAULT_FOLDER}"
     # Update branch and tag metadata
-    git fetch --tags --depth 1 origin "${VAULT_VERSION}"
+    git fetch --depth 1 origin ${CHECKOUT_ARGS} "${VAULT_VERSION}"
```

Comment:
Some nitpicking here to fix some shellcheck linting and how the command is buildup.

The `CHECKOUT_ARGS` is probably better to be an array. And using `CHECKOUT_ARGS` within it self is not needed, as nothing else will be able to have something added before.

Also, all options should be added before `origin` or any other specific repo assignment.


```suggestion
CHECKOUT_ARGS=()
if [ "${CHECKOUT_TAGS}" = "true" ]; then
    CHECKOUT_ARGS+=(--tags)
fi

# Checkout the request
pushd "${VAULT_FOLDER}"
    # Update branch and tag metadata
    git fetch "${CHECKOUT_ARGS[@]}" --depth 1 origin "${VAULT_VERSION}"
```

Reply:
I set it within itself as an easy way to inject additional args if needed, forgot to add the `:-` in case `set -u` is ever used.  Switching to an array make injection impossible and things more complicated.

Will fix the order.